tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -592,6 +592,44 @@ TEST (testTensorTransform, arithmeticProperties11_n)
   gst_harness_teardown (h);
 }
 
+/**
+ * @brief Test for invalid properties of tensor_transform
+ */
+TEST (testTensorTransform, arithmeticProperties12_n)
+{
+  GstHarness *h;
+  gchar *str = NULL;
+
+  h = gst_harness_new ("tensor_transform");
+  ASSERT_TRUE (NULL != h);
+
+  g_object_set (h->element, "mode", GTT_ARITHMETIC, "option", "per-channel:false", NULL);
+  g_object_get (h->element, "option", &str, NULL);
+  g_critical ("option: %s", str);
+  EXPECT_TRUE (str == NULL);
```

Comment:
Isn't it?
EXPECT_TRUE (str != NULL);

Reply:
"per-channel:false" is not valid option. `add|mul|div:...` should be followed. And `str` remains NULL.